From 5b8b6c5933599642e2be322bdc3e081e7c981c3c Mon Sep 17 00:00:00 2001
From: Todd Whiteman <todd.whiteman@joyent.com>
Date: Mon, 31 Jul 2017 12:44:26 -0700
Subject: [PATCH] DOCKER-1081 test api-images-v1-v2.test.js failing due to
 wrong imgapi image layer Reviewed by: Trent Mick <trentm@gmail.com> Approved
 by: Trent Mick <trentm@gmail.com>

---
 test/integration/api-images-v1-v2.test.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/integration/api-images-v1-v2.test.js b/test/integration/api-images-v1-v2.test.js
index dd4e527..280ac89 100644
--- a/test/integration/api-images-v1-v2.test.js
+++ b/test/integration/api-images-v1-v2.test.js
@@ -230,7 +230,7 @@ test('init docker images', function (tt) {
                     + contentLayers.length));
                 return;
             }
-            var digestChain = manifest.layers.slice(0, layerIdx).map(
+            var digestChain = manifest.layers.slice(0, layerIdx+1).map(
                 function (layer) {
                     return layer.digest;
                 }
-- 
2.21.0

